eng/pipelines/templates/jobs/ci.yml

```diff
@@ -66,72 +66,104 @@ parameters:
     default: false
 
 jobs:
-  
+
   - ${{ if eq(parameters['AdvancedBuild'], false) }}:
     - job: 'Build'
       timeoutInMinutes: 90
 
       pool:
         name: azsdk-pool-mms-ubuntu-2004-general
-        vmImage: MMSUbuntu20.04
+        image: azsdk-pool-mms-ubuntu-2004-1espt
+        os: linux
 
       steps:
-      - template: /eng/pipelines/templates/steps/targeting-string-resolve.yml
+      - template: /eng/pipelines/templates/steps/targeting-string-resolve.yml@self
```

Comment:
Why is @self needed here?

Reply:
Since when does the `azure-sdk` account leave PR review comments? 😆

Reply:
I know right? 😂

Reply:
All the @self will be cleaned up from here though regardless 👍

Reply:
Is this you @weshaggard? We probably don't need self at the job level.

Reply:
lol... I confess this was me :)

I didn't post as @self :)